nap/utils.py

```diff
@@ -42,13 +41,20 @@ def flatten_params(k, vs):
 
         flat_params = [
             flatten_params(k, v)
-            for (k, v) in params.iteritems()
+            for (k, v) in params.items()
         ]
 
         # since we can have more than one value for a single key, we use a
         # tuple of two tuples instead of a dictionary
         params_tuple = tuple(itertools.chain(*flat_params))
-        param_string = urllib.urlencode(params_tuple)
+        param_string = urllib.parse.urlencode(params_tuple)
```

Comment:
Hooray, I can. That should actually be this:

from six.moves.urllib.parse import urlencode

I'll scour the changes to see what else I might have missed.

Thanks for taking a look!


On Fri, Nov 13, 2015 at 9:22 AM, Joseph Bane <notifications@github.com>
wrote:

> In nap/utils.py
> <https://github.com/CashStar/nap/pull/9#discussion_r44786314>:
>
> >          ]
> >
> >          # since we can have more than one value for a single key, we use a
> >          # tuple of two tuples instead of a dictionary
> >          params_tuple = tuple(itertools.chain(*flat_params))
> > -        param_string = urllib.urlencode(params_tuple)
> > +        param_string = urllib.parse.urlencode(params_tuple)
>
> Could you please elaborate on this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CashStar/nap/pull/9/files#r44786314>.
>